components/ReportsLists/index.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+
+const H3 = styled.h3`
+  margin-bottom:1px;
+  padding:5px;
+`;
+const P = styled.p`
+  margin:3px;
+  padding:1px;
+`;
+const H2 = styled.h2`
```

Comment:
Please don't abbreviate names for styled components, future Hristo can thank me later.